Error running Solo with brew install command

### Describe the bug

Previously deployed Solo. Follow these instructions: https://solo.hiero.org/main/docs/step-by-step-guide/. Installs an older version of Solo that I had previously (0.48) and not the latest.

Also encountered error:

Error deploying Solo in one-shot mode: failed to upgrade chart haproxy-ingress-1: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5 --install --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"

### Describe the expected behavior

Should install the latest version.

### To Reproduce

Install and older version of Solo. Then use the brew command to install the latest.

### Additional Context

(base) user@JJRWPX6DLV ~ % brew install hiero-ledger/tools/solo

==> Auto-updating Homebrew...
Adjust how often this is run with `$HOMEBREW_AUTO_UPDATE_SECS` or disable with
`$HOMEBREW_NO_AUTO_UPDATE=1`. Hide these hints with `$HOMEBREW_NO_ENV_HINTS=1` (see `man brew`).
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
go-air: Live reload for Go apps
jqfmt: Opinionated formatter for jq
mac-cleanup-go: TUI macOS cleaner that scans caches/logs and lets you select what to delete
==> New Casks
codex-app: OpenAI's Codex desktop app for managing coding agents
codexbar: Menu bar usage monitor for Codex and Claude
commander: AI agent operator
elegoo-slicer: Open-source slicer for FDM 3D printers
ethui: Ethereum development toolkit with wallet and anvil support
middledrag: Middle-click and middle-drag via three-finger trackpad gestures
plasticity: 3D modeling software for concept artists and designers
repobar: Menu bar dashboard for GitHub repository health
retrace: Local-first screen recording and search application
trimmy: Paste-once, run-once clipboard cleaner for terminal snippets
yandextelemost: Yandex video calls and meetings platform

You have 66 outdated formulae and 2 outdated casks installed.

==> Tapping hiero-ledger/tools
Cloning into '/opt/homebrew/Library/Taps/hiero-ledger/homebrew-tools'...
remote: Enumerating objects: 168, done.
remote: Counting objects: 100% (71/71), done.
remote: Compressing objects: 100% (57/57), done.
remote: Total 168 (delta 21), reused 40 (delta 14), pack-reused 97 (from 1)
Receiving objects: 100% (168/168), 64.10 KiB | 713.00 KiB/s, done.
Resolving deltas: 100% (40/40), done.
Tapped 9 formulae (23 files, 91.7KB).
==> Fetching downloads for: solo
✔︎ Bottle Manifest fmt (12.1.0)                                                              Downloaded    7.3KB/  7.3KB
✔︎ Bottle fmt (12.1.0)                                                                       Downloaded  291.8KB/291.8KB
✔︎ Bottle Manifest ada-url (3.4.2)                                                           Downloaded    8.5KB/  8.5KB
✔︎ Bottle ada-url (3.4.2)                                                                    Downloaded  348.7KB/348.7KB
✔︎ Bottle Manifest brotli (1.2.0)                                                            Downloaded    8.0KB/  8.0KB
✔︎ Bottle brotli (1.2.0)                                                                     Downloaded  794.0KB/794.0KB
✔︎ Bottle Manifest c-ares (1.34.6)                                                           Downloaded    7.5KB/  7.5KB
✔︎ Bottle c-ares (1.34.6)                                                                    Downloaded  304.4KB/304.4KB
✔︎ Bottle Manifest hdrhistogram_c (0.11.9)                                                   Downloaded    7.7KB/  7.7KB
✔︎ Bottle hdrhistogram_c (0.11.9)                                                            Downloaded   43.8KB/ 43.8KB
✔︎ Bottle Manifest icu4c@78 (78.2)                                                           Downloaded    9.7KB/  9.7KB
✔︎ Bottle Manifest libnghttp2 (1.68.0)                                                       Downloaded    7.3KB/  7.3KB
✔︎ Bottle libnghttp2 (1.68.0)                                                                Downloaded  221.5KB/221.5KB
✔︎ Bottle Manifest libnghttp3 (1.15.0)                                                       Downloaded    7.3KB/  7.3KB
✔︎ Bottle libnghttp3 (1.15.0)                                                                Downloaded  189.8KB/189.8KB
✔︎ Bottle Manifest ca-certificates (2025-12-02)                                              Downloaded    2.0KB/  2.0KB
✔︎ Bottle ca-certificates (2025-12-02)                                                       Downloaded  131.8KB/131.8KB
✔︎ Bottle Manifest openssl@3 (3.6.1)                                                         Downloaded   11.8KB/ 11.8KB
✔︎ Bottle Manifest libngtcp2 (1.20.0)                                                        Downloaded    9.3KB/  9.3KB
✔︎ Bottle Manifest libuv (1.51.0)                                                            Downloaded    9.6KB/  9.6KB
✔︎ Bottle Manifest llhttp (9.3.0)                                                            Downloaded    7.2KB/  7.2KB
✔︎ Bottle Manifest simdjson (4.2.4)                                                          Downloaded    7.4KB/  7.4KB
✔︎ Bottle libngtcp2 (1.20.0)                                                                 Downloaded  399.2KB/399.2KB
✔︎ Bottle libuv (1.51.0)                                                                     Downloaded  370.0KB/370.0KB
✔︎ Bottle llhttp (9.3.0)                                                                     Downloaded   36.4KB/ 36.4KB
✔︎ Bottle Manifest sqlite (3.51.2)                                                           Downloaded   11.4KB/ 11.4KB
✔︎ Bottle Manifest uvwasi (0.0.23)                                                           Downloaded    8.3KB/  8.3KB
✔︎ Bottle simdjson (4.2.4)                                                                   Downloaded    1.2MB/  1.2MB
✔︎ Bottle Manifest xz (5.8.2)                                                                Downloaded   11.8KB/ 11.8KB
✔︎ Formula solo (0.54.0)                                                                     Verified      3.0MB/  3.0MB
✔︎ Bottle Manifest node (25.5.0)                                                             Downloaded   27.5KB/ 27.5KB
✔︎ Bottle sqlite (3.51.2)                                                                    Downloaded    2.4MB/  2.4MB
✔︎ Bottle uvwasi (0.0.23)                                                                    Downloaded   72.5KB/ 72.5KB
✔︎ Bottle xz (5.8.2)                                                                         Downloaded  766.0KB/766.0KB
✔︎ Bottle icu4c@78 (78.2)                                                                    Downloaded   32.0MB/ 32.0MB
✔︎ Bottle node (25.5.0)                                                                      Downloaded   19.1MB/ 19.1MB
✔︎ Bottle openssl@3 (3.6.1)                                                                  Downloaded   10.9MB/ 10.9MB
==> Installing solo from hiero-ledger/tools
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Settings.

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 16.2.



This is a Tier 2 configuration:
  https://docs.brew.sh/Support-Tiers#tier-2
You can report Tier 2 unrelated issues to Homebrew/* repositories!
Read the above document before opening any issues or PRs.

==> Installing dependencies for hiero-ledger/tools/solo: fmt, ada-url, brotli, c-ares, hdrhistogram_c, icu4c@78, libnghttp2, libnghttp3, ca-certificates, openssl@3, libngtcp2, libuv, llhttp, simdjson, sqlite, uvwasi, xz and node
==> Installing hiero-ledger/tools/solo dependency: fmt
==> Pouring fmt--12.1.0.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/fmt/12.1.0: 29 files, 1.1MB
==> Installing hiero-ledger/tools/solo dependency: ada-url
==> Pouring ada-url--3.4.2.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/ada-url/3.4.2: 55 files, 1.3MB
==> Installing hiero-ledger/tools/solo dependency: brotli
==> Pouring brotli--1.2.0.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/brotli/1.2.0: 33 files, 1.9MB
==> Installing hiero-ledger/tools/solo dependency: c-ares
==> Pouring c-ares--1.34.6.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/c-ares/1.34.6: 176 files, 1MB
==> Installing hiero-ledger/tools/solo dependency: hdrhistogram_c
==> Pouring hdrhistogram_c--0.11.9.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/hdrhistogram_c/0.11.9: 22 files, 173.4KB
==> Installing hiero-ledger/tools/solo dependency: icu4c@78
==> Pouring icu4c@78--78.2.arm64_sonoma.bottle.1.tar.gz
🍺  /opt/homebrew/Cellar/icu4c@78/78.2: 279 files, 88.0MB
==> Installing hiero-ledger/tools/solo dependency: libnghttp2
==> Pouring libnghttp2--1.68.0.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/libnghttp2/1.68.0: 14 files, 790.6KB
==> Installing hiero-ledger/tools/solo dependency: libnghttp3
==> Pouring libnghttp3--1.15.0.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/libnghttp3/1.15.0: 20 files, 606.6KB
==> Installing hiero-ledger/tools/solo dependency: ca-certificates
==> Pouring ca-certificates--2025-12-02.all.bottle.1.tar.gz
==> Regenerating CA certificate bundle from keychain, this may take a while...
🍺  /opt/homebrew/Cellar/ca-certificates/2025-12-02: 4 files, 236.4KB
==> Installing hiero-ledger/tools/solo dependency: openssl@3
==> Pouring openssl@3--3.6.1.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/openssl@3/3.6.1: 7,624 files, 37.5MB
==> Installing hiero-ledger/tools/solo dependency: libngtcp2
==> Pouring libngtcp2--1.20.0.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/libngtcp2/1.20.0: 21 files, 1.3MB
==> Installing hiero-ledger/tools/solo dependency: libuv
==> Pouring libuv--1.51.0.arm64_sonoma.bottle.1.tar.gz
🍺  /opt/homebrew/Cellar/libuv/1.51.0: 34 files, 1.3MB
==> Installing hiero-ledger/tools/solo dependency: llhttp
==> Pouring llhttp--9.3.0.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/llhttp/9.3.0: 13 files, 160.4KB
==> Installing hiero-ledger/tools/solo dependency: simdjson
==> Pouring simdjson--4.2.4.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/simdjson/4.2.4: 19 files, 6.6MB
==> Installing hiero-ledger/tools/solo dependency: sqlite
==> Pouring sqlite--3.51.2.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/sqlite/3.51.2: 13 files, 5.3MB
==> Installing hiero-ledger/tools/solo dependency: uvwasi
==> Pouring uvwasi--0.0.23.arm64_sonoma.bottle.1.tar.gz
🍺  /opt/homebrew/Cellar/uvwasi/0.0.23: 15 files, 311KB
==> Installing hiero-ledger/tools/solo dependency: xz
==> Pouring xz--5.8.2.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/xz/5.8.2: 96 files, 2.7MB
==> Installing hiero-ledger/tools/solo dependency: node
==> Pouring node--25.5.0.arm64_sonoma.bottle.tar.gz
🍺  /opt/homebrew/Cellar/node/25.5.0: 1,932 files, 79.3MB
==> Installing hiero-ledger/tools/solo
==> npm install --loglevel=silly --global --build-from-source --cache=/Users/user/Library/Caches/Homebrew/npm_cache --p
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /opt/homebrew
Could not symlink bin/solo
Target /opt/homebrew/bin/solo
already exists. You may want to remove it:
  rm '/opt/homebrew/bin/solo'

To force the link and overwrite all conflicting files:
  brew link --overwrite solo

To list all files that would be deleted:
  brew link --overwrite solo --dry-run

Possible conflicting files are:
/opt/homebrew/bin/solo -> /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/solo.js
==> Summary
🍺  /opt/homebrew/Cellar/solo/0.54.0: 27,049 files, 261.4MB, built in 38 seconds
==> Running `brew cleanup solo`...
Disable this behaviour by setting `HOMEBREW_NO_INSTALL_CLEANUP=1`.
Hide these hints with `HOMEBREW_NO_ENV_HINTS=1` (see `man brew`).
(base) user@JJRWPX6DLV ~ % solo --version


******************************* Solo *********************************************
Version			: 0.48.0
**********************************************************************************
(node:40518) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
(base) user@JJRWPX6DLV ~ % solo one-shot single deploy


******************************* Solo *********************************************
Version			: 0.48.0
Kubernetes Context	: kind-solo
Kubernetes Cluster	: kind-solo
Current Command		: one-shot single deploy
**********************************************************************************
✔ Check dependencies [0.8s]
  ✔ Check dependency: helm [OS: darwin, Release: 23.5.0, Arch: arm64] [0.8s]
  ✔ Check dependency: kubectl [OS: darwin, Release: 23.5.0, Arch: arm64] [0.8s]
✔ Setup chart manager [6s]
↓ Install Kind
↓ Create default cluster

(node:40924) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
    ✔ Check Envoy Proxy for: node1, cluster: solo-5330225f
    ✔ Check Envoy Proxy for: node1, cluster: solo-5330225f
✔ Initialize
✔ solo cluster-ref config connect
  ✔ Initialize
  ✔ solo-5330225f
  ✔ Test connection to cluster: solo-5330225f
  ✔ Associate a context with a cluster reference: solo-5330225f
✔ solo deployment config create
  ✔ Initialize
  ✔ Adding deployment: solo-deployment-5330225f with namespace: solo-5330225f to local config
✔ solo deployment cluster attach [0.1s]
  ✔ Initialize
  ✔ Verify args
  ✔ check ledger phase
  ✔ Test cluster connection: solo-5330225f, context: kind-solo
  ✔ Verify prerequisites
  ✔ add cluster-ref: solo-5330225f for deployment: solo-deployment-5330225f in local config
  ✔ create remote config for deployment: solo-deployment-5330225f in cluster: solo-5330225f
✔ solo cluster-ref config setup [0.3s]
  ✔ Initialize
  ✔ Install cluster charts [0.2s]
    ✔ Install pod-monitor-role ClusterRole
    ✔ Install MinIO Operator chart
    ✔ Install Prometheus Stack chart
✔ solo keys consensus generate [0.8s]
  ✔ Initialize
  ✔ Generate gossip keys
    ✔ Backup old files
    ✔ Gossip key for node: node1
  ✔ Generate gRPC TLS Keys [0.6s]
    ✔ Backup old files
    ✔ TLS key for node: node1 [0.6s]
  ✔ Finalize
↓ solo block node add
✔ solo consensus network deploy [19s]
  ✔ Initialize [0.1s]
    ✔ Acquire lock - lock acquired successfully, attempt: 1/10
  ↓ Copy gRPC TLS Certificates
  ✔ Prepare staging directory
    ✔ Copy Gossip keys to staging
    ✔ Copy gRPC TLS keys to staging
  ✔ Copy node keys to secrets
    ✔ Copy TLS keys
    ✔ Node: node1, cluster: kind-solo
      ✔ Copy Gossip keys
  ✔ Install chart 'solo-deployment' [1s]
  ↓ Check for load balancer
  ↓ Redeploy chart with external IP address config
  ✔ Check node pods are running [9s]
    ✔ Check Node: node1, Cluster: solo-5330225f [9s]
  ✔ Check proxy pods are running
    ✔ Check HAProxy for: node1, cluster: solo-5330225f
    ✔ Check Envoy Proxy for: node1, cluster: solo-5330225f
  ✔ Check auxiliary pods are ready [8s]
    ✔ Check MinIO [8s]
  ✔ Add node and proxies to remote config
  ↓ Copy block-nodes.json
✔ solo consensus node setup [5s]
  ✔ Load configuration
  ✔ Initialize
  ✔ Validate nodes states
    ✔ Validating state for node node1 - valid state: requested
  ✔ Identify network pods
    ✔ Check network pod: node1
  ✔ Fetch platform software into network nodes [4s]
    ✔ Update node: node1 [ platformVersion = v0.66.0, context = kind-solo ] [4s]
  ✔ Setup network nodes [0.4s]
    ✔ Node: node1 [0.3s]
      ✔ Copy configuration files [0.1s]
      ✔ Set file permissions [0.2s]
  ✔ setup network node folders
  ✔ Change node state to configured in remote config
✔ solo consensus node start [51s]
  ✔ Load configuration
  ✔ Initialize
  ✔ Validate nodes states
    ✔ Validating state for node node1 - valid state: configured
  ✔ Identify existing network nodes
    ✔ Check network pod: node1
  ↓ Upload state files network nodes
  ✔ Starting nodes [0.1s]
    ✔ Start node: node1
  ✔ Enable port forwarding for debug port and/or GRPC port
  ✔ Check all nodes are ACTIVE [20s]
    ✔ Check network pod: node1  - status ACTIVE, attempt: 17/300 [20s]
  ✔ Check node proxies are ACTIVE [22s]
    ✔ Check proxy for node: node1 [22s]
  ✔ Change node state to started in remote config
  ✔ Add node stakes [5s]
    ✔ Adding stake for node: node1 [5s]
  ✔ set gRPC Web endpoint [2s]
❯ solo mirror node add
  ✔ Initialize [0.3s]
    ✔ Acquire lock - lock acquired successfully, attempt: 1/10
  ❯ Enable mirror-node
    ✔ Prepare address book [0.1s]
    ✖ Install mirror ingress controller
    ◼ Deploy mirror-node
  ◼ Check pods are ready
  ◼ Seed DB data
  ◼ Add mirror node to remote config
  ◼ Enable port forwarding for mirror ingress controller
◼ solo explorer node add
◼ solo relay node add
◼ Create Accounts
◼ Finish


 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]
Skipping Grafana Agent chart installation

 *** Installed Charts ***
-------------------------------------------------------------------------------
 - kube-prometheus-stack [kube-prometheus-stack-52.0.1]
 - operator [operator-7.1.1]
⏭️  ClusterRole pod-monitor-role already exists, skipping
⏭️  MinIO Operator chart already installed, skipping
⏭️  Prometheus Stack chart already installed, skipping

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]
 - Installed solo-deployment chart, version: 0.57.0

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]
Using requested port 50211
Using requested port 30212

 *** Home Directories ***
-------------------------------------------------------------------------------
 - /Users/user/.solo
 - /Users/user/.solo/logs
 - /Users/user/.solo/cache
 - /Users/user/.solo/cache/values-files

 *** Chart Repository ***
-------------------------------------------------------------------------------
[ None ]
Stopping port-forwarder for port [30212]
*********************************** ERROR *****************************************
Error deploying Solo in one-shot mode: failed to upgrade chart haproxy-ingress-1: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
SoloError: Error deploying Solo in one-shot mode: failed to upgrade chart haproxy-ingress-1: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
    at DefaultOneShotCommand.deployInternal (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/commands/one-shot/default-one-shot.ts:507:13)
    at Object.handler (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/core/command-path-builders/command-builder.ts:120:47)
Caused by: SoloError: failed to upgrade chart haproxy-ingress-1: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
    at ChartManager.upgrade (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/core/chart-manager.ts:200:13)
    at Task.task [as taskFn] (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/commands/mirror-node.ts:603:17)
    at Task.run (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/node_modules/listr2/dist/index.js:1905:5)
Caused by: HelmExecutionException: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
    at ChildProcess.<anonymous> (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/integration/helm/execution/helm-execution.ts:79:13)
    at ChildProcess.emit (node:events:508:20)

  Caused by: failed to upgrade chart haproxy-ingress-1: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
  SoloError: failed to upgrade chart haproxy-ingress-1: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
    at ChartManager.upgrade (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/core/chart-manager.ts:200:13)
    at Task.task [as taskFn] (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/commands/mirror-node.ts:603:17)
    at Task.run (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/node_modules/listr2/dist/index.js:1905:5)
Caused by: HelmExecutionException: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
    at ChildProcess.<anonymous> (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/integration/helm/execution/helm-execution.ts:79:13)
    at ChildProcess.emit (node:events:508:20)

    Caused by: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
    HelmExecutionException: Helm command failed with exit code 1. Command: '/Users/user/.solo/bin/helm upgrade --reuse-values --output json --namespace solo-5330225f --kube-context kind-solo --version 0.14.5  --install  --values /opt/homebrew/lib/node_modules/@hashgraph/solo/dist/resources/ingress-controller-values.yaml --set fullnameOverride=mirror-ingress-controller --set controller.ingressClass=mirror-ingress-class --set controller.extraArgs.controller-class=mirror-ingress-controller haproxy-ingress-1 haproxy-ingress/haproxy-ingress'. Error: Error: Unable to continue with install: ClusterRole "mirror-ingress-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "solo-5330225f": current value is "solo-4cdf4eaf"
    at ChildProcess.<anonymous> (file:///opt/homebrew/lib/node_modules/@hashgraph/solo/src/integration/helm/execution/helm-execution.ts:79:13)
    at ChildProcess.emit (node:events:508:20)

***********************************************************************************

--- a/docs/site/content/en/templates/step-by-step-guide.template.md
+++ b/docs/site/content/en/templates/step-by-step-guide.template.md
@@ -39,7 +39,7 @@ Choose your platform below:
 # Docker Desktop > Settings > Resources > Memory
 
 # 3. Remove existing npm based installs
-[[ "$(command -v npm >/dev/null 2>&1 && echo 0 || echo 1)" -eq 0 ]] && { npm uninstall -g @hashgraph/solo >/dev/null 2>&1 || /bin/true }
+\[\[ "$(command -v npm >/dev/null 2>&1 && echo 0 || echo 1)" -eq 0 \]\] && { npm uninstall -g @hashgraph/solo >/dev/null 2>&1 || /bin/true }
 
 # 4. Install Solo (this installs all other dependencies automatically)
 brew tap hiero-ledger/tools
@@ -74,7 +74,15 @@ curl -fsSLo kubectl "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/re
 chmod +x kubectl
 sudo mv kubectl /usr/local/bin/kubectl
 
-# 4. Install Solo (this installs remaining dependencies automatically)
+# 4. Remove existing npm based installs
+\[\[ "$(command -v npm >/dev/null 2>&1 && echo 0 || echo 1)" -eq 0 \]\] && { npm uninstall -g @hashgraph/solo >/dev/null 2>&1 || /bin/true }
+
+# 5. Install Solo (this installs all other dependencies automatically)
+brew tap hiero-ledger/tools
+brew update
+brew install solo
+
+# 6. Install Solo (this installs remaining dependencies automatically)
 brew install hiero-ledger/tools/solo
 
 # Verify the installation
@@ -106,7 +114,15 @@ curl -fsSLo kubectl "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/re
 chmod +x kubectl
 sudo mv kubectl /usr/local/bin/kubectl
 
-# 4. Install Solo (this installs remaining dependencies automatically)
+# 4. Remove existing npm based installs
+\[\[ "$(command -v npm >/dev/null 2>&1 && echo 0 || echo 1)" -eq 0 \]\] && { npm uninstall -g @hashgraph/solo >/dev/null 2>&1 || /bin/true }
+
+# 5. Install Solo (this installs all other dependencies automatically)
+brew tap hiero-ledger/tools
+brew update
+brew install solo
+
+# 6. Install Solo (this installs remaining dependencies automatically)
 brew install hiero-ledger/tools/solo
 
 # Verify the installation

